elasticsearch/includes/classes/class-elasticsearch.php

```diff
@@ -84,12 +84,19 @@ public function filter__ep_index_name( $index_name, $blog_id, $indexables ) {
 
 	public function filter__ep_do_intercept_request( $request, $query, $args, $failures ) {
 		$fallback_error = new \WP_Error( 'vip-elasticsearch-upstream-request-failed', 'There was an error connecting to the upstream Elasticsearch server' );
+		$timeout = 1;
 
 		// TEMP - currently ES server is using a self signed cert during the testing phase...that'll be changed in the near
 		// future, at which time we can remove this
 		$args['sslverify'] = false;
-	
-		$request = vip_safe_wp_remote_request( $query['url'], $fallback_error, 3, 1, 20, $args );
+
+		// If query url ends with '_bulk'
+		if ( 0 === substr_compare( $query[ 'url' ], '_bulk', -5 ) ) {
```

Comment:
This won't account for query params and such - it would be better to use `parse_url()` to get the actual path, then check it for the expected endpoint.

Reply:
Good idea!